var/spack/repos/builtin/packages/genomeworks/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Genomeworks(CMakePackage, CudaPackage):
+    """SDK for GPU accelerated genome assembly and analysis."""
+
+    homepage = "https://clara-parabricks.github.io/GenomeWorks/"
+    url      = "https://github.com/clara-parabricks/GenomeWorks/archive/v0.5.3.tar.gz"
+    git      = "https://github.com/clara-parabricks/GenomeWorks.git"
+
+    version('0.5.3', tag='v0.5.3', submodules=True)
+    version('0.5.2', tag='v0.5.2', submodules=True)
+    version('0.5.1', tag='v0.5.1', submodules=True)
+    version('0.5.0-rc1', tag='v0.5.0-rc1', submodules=True)
```

Comment:
We don't usually add release candidates, so I would:
```suggestion
```

Reply:
No reason. Removing.